tests/Extensions/Abstractions.UnitTests/EventSourcesEventIdsTests.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Linq;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Omex.Extensions.Abstractions.UnitTests
+{
+	[TestClass]
+	public class EventSourcesEventIdsTests
+	{
+		[TestMethod]
+		public void EventIdsUnique()
+		{
+			int[] values = Enum.GetValues(typeof(EventSourcesEventIds)).Cast<int>().ToArray();
```

Comment:
Do you need `.ToArray()`? Also `.ToList()` might be more efficient if resizing occurs

Reply:
Update to don't use `ToArray()`, thanks